Update ASEs Change App Service Plan requirements

Daniellezha opened the pull request:
'Move an app to another App Service plan'  -  An App Service Environment can have multiple webspaces now, your app can only move between plans that are created in the same webspace.

Daniellezha committed:
Update ASEs Change App Service Plan requirements

'Move an app to another App Service plan'  -  An App Service Environment can have multiple webspaces now, your app can only move between plans that are created in the same webspace.

cephalin commented on articles/app-service/app-service-plan-manage.md:
```diff
@@ -42,7 +42,7 @@ You can create an empty App Service plan, or you can create a plan as part of ap
 
 > [!NOTE]
-> Azure deploys each new App Service plan into a deployment unit, internally called a webspace. Each region can have many webspaces, but your app can only move between plans that are created in the same webspace. An App Service Environment is an isolated webspace, so apps can be moved between plans in the same App Service Environment, but not between plans in different App Service Environments.
+> Azure deploys each new App Service plan into a deployment unit, internally called a webspace. Each region can have many webspaces, but your app can only move between plans that are created in the same webspace. An App Service Environment can have multiple webspaces, your app can only move between plans that are created in the same webspace.
```
"An App Service Environment can have multiple webspaces, **but** your app can only move between plans that are created in the same webspace."

Daniellezha committed:
Update app-service-plan-manage.md

update text to  "An App Service Environment can have multiple webspaces, but your app can only move between plans that are created in the same webspace"
